PWGCF/TwoParticleCorrelations/Tasks/CorrSparse.cxx

```diff
@@ -110,20 +119,17 @@ struct CorrSparse {
     }
   }
 
-  // make the filters and cuts. I was told not to include chi2 since its not needed for run 3 data.
+  // make the filters and cuts.
 
   Filter collisionFilter = nabs(aod::collision::posZ) < cfgZVtxCut;
 
-  Filter trackFilter = (nabs(aod::track::eta) < cfgEtaCut) && (aod::track::pt > cfgPtCutMin) && (aod::track::pt < cfgPtCutMax) && ((requireGlobalTrackInFilter()) || (aod::track::isGlobalTrackSDD == (uint8_t) true))
-                       //&& (aod::track::tpcChi2NCl < cfgCutChi2prTPCcls)
-                       && (nabs(aod::track::dcaZ) > cfgDCAzCut) && (cfgDCAxySigmaCut * (0.0015f + 0.005f / npow(aod::track::pt, 1.1f)) < nabs(aod::track::dcaXY));
-  //
+  Filter trackFilter = (nabs(aod::track::eta) < cfgEtaCut) && (aod::track::pt > cfgPtCutMin) && (aod::track::pt < cfgPtCutMax) && ((requireGlobalTrackInFilter()) || (aod::track::isGlobalTrackSDD == (uint8_t) true));
 
   // define the filtered collisions and tracks
+
   using aodCollisions = soa::Filtered<soa::Join<aod::Collisions, aod::EvSels, aod::CentFT0Cs>>;
```

Comment:
For this one, as suggested, you have to use `UpperCamelCase`, i.e. `aodCollisions`
And similar for the similar ones